onnx/defs/generator/defs.cc

```diff
@@ -51,6 +98,60 @@ ONNX_OPERATOR_SET_SCHEMA(
             return;
           }
 
+          if (nullptr != value_int) {
+            // OpSchema::Verify check ensures that the attribute value has_i():
+            if (!value_int->has_i())
+              fail_shape_inference("Attribute 'value_int' expect an integer.")
+            updateOutputElemType(ctx, 0, TensorProto::INT64);
+            updateOutputShape(ctx, 0, TensorShapeProto());
+            return;
+          }
+
+          if (nullptr != value_ints) {
+            // OpSchema::Verify check ensures that the attribute value has ints.
+            if (value_ints->ints_size() < 1)
+              fail_shape_inference("Attribute 'value_ints' expect a list of integers.")
+            updateOutputElemType(ctx, 0, TensorProto::INT64);
+            appendDim(getOutputShape(ctx, 0), value_ints->ints_size());
```

Comment:
May be slightly simpler to write "updateOutputShape(ctx, 0, {value_ints->ints_size()})". Same comment for other statements below.

Reply:
This syntax do not compile. The compiler don't know how to create a `TensorShapeProto` from this vector.

Reply:
Sorry, please ignore my suggestion. It needs to be a bit more complicated, needs a Dim, not integer value.